### PR TITLE
Update licenses for Garnet/FXL renaming

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

**Comment by jason-simmons** [comment]
@Hixie @kulakowski @abarth

**Comment by abarth** [comment]
LGTM

File: tools/licenses/lib/main.dart [hunk 1]
```diff
@@ -2150,6 +2150,10 @@ class RepositoryGarnetDirectory extends RepositoryDirectory {
   @override
   bool shouldRecurse(fs.IoNode entry) {
     return entry.name != 'bin'
+        && entry.name != 'docs'
+        && entry.name != 'examples'
+        && entry.name != 'go'
+        && entry.name != 'lib'
```
Comment (Hixie) [inline tools/licenses/lib/main.dart]: lib isn't shipped?

Comment (jason-simmons) [reply tools/licenses/lib/main.dart]: The libraries that we link to are in garnet/public/lib

Comment (abarth) [reply tools/licenses/lib/main.dart]: `//garnet/lib` is meant to be private.  We shouldn't have dependencies on anything outside the `//garnet/public` directory (and, transitively, `public` shouldn't have any such dependencies either).

Comment (Hixie) [reply tools/licenses/lib/main.dart]: Private vs public isn't the issue here, it's whether any of the files in `lib/` in any way influence the contents of the final binary.

Comment (abarth) [reply tools/licenses/lib/main.dart]: `public` doesn't have any dependencies in `//garnet` on anything outside of `public`.  In particular, `public` doesn't depend on `lib` or `bin`.

**Pull request merged**